packages/pycharm-community-bin/PyCharm.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Icon=/usr/src/pacstall/pycharm-community-bin/usr/bin/pycharm-community/bin/pycharm.svg
+Name=PyCharm Community Edition
+Exec=/usr/src/pacstall/pycharm-community-bin/usr/bin/pycharm-community/bin/pycharm.sh
```

Comment:
You should strip `/usr/src/pacstall/pycharm-community-bin` from everything, as that will be symlinked to `/`, so this line, for instance, should be: `/usr/bin/pycharm-community/bin/pycharm.sh`